packages/kolibri-common/composables/useCoachMetadataTags.js

```diff
@@ -0,0 +1,73 @@
+import { ref } from 'kolibri.lib.vueCompositionApi';
+import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+import coreStrings from 'kolibri.utils.coreStrings';
+import commonCoreStringsMixin from 'kolibri.coreVue.mixins.commonCoreStrings';
+
+export function useCoachMetadataTags(contentNode) {
+  const tags = ref([]);
+
+  const getCategoryTags = () => {
+    if (!contentNode.categories) return [];
+    return contentNode.categories.map(category =>
+      commonCoreStringsMixin.methods.coreString(category),
+    );
+  };
+
+  const getLevelTags = () => {
+    if (!contentNode.grade_levels) return [];
+    return contentNode.grade_levels.map(grade_levels =>
+      commonCoreStringsMixin.methods.coreString(grade_levels),
+    );
+  };
+
+  const getLanguageTag = () => {
+    if (!contentNode.lang) return [];
+    return contentNode.lang.length > 1 ? ['Multiple languages'] : [contentNode.lang.lang_name];
```

Comment:
How are we going to use this array w/ the `'Multiple languages'` string?

Reply:
I'm also generally just a little unsure about this length check - the `lang` attribute of the `contentNode` is always an object, indicating the language of the contentNode: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/content/api.py#L758

So, this length check will always be false anyway.

Reply:
Thanks for catching that! You’re spot on—the` contentNode.lang `is always an object here, so checking the length is pretty pointless. I've tweaked the logic to handle the lang attribute correctly. So since we’re only working with one language for each contentNode, I have removed  the length check and now it simply returns the language name.